perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -335,18 +338,17 @@ sub garden_renew_as_new {
 
 =head2 waste_sub_due
 
-Returns true/false if now is less than garden_due_days before DATE.
+Returns true/false if now is after garden_due_date.
 
 =cut
 
 sub waste_sub_due {
-    my ( $self, $date ) = @_;
+    my ($self, $date) = @_;
 
-    my $now = DateTime->now->set_time_zone( FixMyStreet->local_time_zone );
+    my $now = DateTime->now->set_time_zone(FixMyStreet->local_time_zone);
     my $sub_end = DateTime::Format::W3CDTF->parse_datetime($date);
-
-    my $diff = $now->delta_days($sub_end)->in_units('days');
-    return $diff <= $self->garden_due_days;
+    my $due_date = $self->garden_due_date($sub_end->truncate(to => 'day'));
+    return $now >= $due_date;
 }
```

Comment:
Some artefacts here from copy and pasting the updated version in the mainline `Garden.pm` which I just decided to leave in.